tests/test_dummy_converter.py

```diff
@@ -62,3 +51,55 @@ def test_default_geo_config_str() -> None:
     1    2    3
     1 1000    0
 """
+
+TEST_DIR = tempfile.TemporaryDirectory()
+
+
+def test_default_beam_config_str() -> None:
+    """Test if the default BeamConfig str representation is correct."""
+    assert str(BeamConfig()) == BEAM_STR
+
+
+def test_default_mat_config_str() -> None:
+    """Test if the default MatConfig str representation is correct."""
+    assert str(MatConfig()) == MAT_STR
+
+
+def test_default_detect_config_str() -> None:
+    """Test if the default MatConfig str representation is correct."""
+    assert str(DetectConfig()) == DETECT_STR
+
+
+def test_default_geo_config_str() -> None:
+    """Test if the default MatConfig str representation is correct."""
+    assert str(GeoConfig()) == GEO_STR
+
+
+def test_parser_run() -> None:
+    """Test if dummy parser generates output files corectly"""
+    dummy_runner = Runner(DummmyParser(), {}, TEST_DIR.name)
+    dummy_runner.run_parser()
```

Comment:
There is no assert here, why ?

Reply:
Is there anything that should be asserted? I assumed we can just check if the parser can run without errors.

Reply:
but how do you know that it runs without errors ?
What do you expect from it when being called with empty input ?
If you expect that no files are generated, then count number of files in some directory and check if it is zero.
If you expect that it generates some files, then check if they exists, have non-zero size, have some meaningful content.
Do you expect it to return some return value ?

Reply:
I think this should be refactored into a fixture, that runs before further tests, rather than being a test itself.

Reply:
Refactored this into a fixture.